blackbox/docs/configuration.txt

```diff
@@ -1865,6 +1865,21 @@ Enterprise License
 
   Setting this to ``false`` disables the `Enterprise Edition`_ of CrateDB.
 
+
+.. _conf_lang_js_enabled:
+
+Javascript Language
+-------------------
+
+
+**lang_js.enabled**
```

Comment:
what about using lang.js.enabled so that other langs can use that settings namespace